packages/components/src/styles/components/layout/grid.scss

```diff
@@ -27,6 +31,41 @@
   gap: var(--hds-layout-grid-row-gap) var(--hds-layout-grid-column-gap);
 }
 
+// responsive column widths (sm, md, lg)
+
+// Note: We repeat the calculation using the view specific variables as
+// resetting just the variable values themselves doesn't work as the inline styles take precedence
+
+// "sm" view, 0–768px (mobile devices)
+.hds-layout-grid--column-width-sm {
+  @include hds-breakpoint-below("md") {
+    grid-template-columns: repeat(
+      var(--hds-layout-grid-column-fill-type-sm),
+      minmax(calc(var(--hds-layout-grid-column-width-sm) - var(--hds-layout-grid-column-gap)), 1fr)
+    );
+  }
+}
+
+// "md" view, 768px–1088px (tablets and small laptops)
+.hds-layout-grid--column-width-md {
```

Comment:
[thought/suggestion] with the current implementation (using "between") the API requires that consumers explicitly declare a value for each breaking point (unless there is some sort of extra "default" value, but I don't think is a viable option); which means, with the likely introduction of `xl`/`xxl` breakpoints too, it becomes a bit verbose. What if instead we use the "above" mixin here? In this way consumers have to declare only a few brekpoints, because implicitly the upper breakpoints are covered, and it's also much easier to think about mobile-first responsiveness